tracks/ruby/exercises/hamming/README.md

```diff
@@ -36,24 +43,21 @@ class Hamming
 end
 ```
 
-I particularly like how the English instructions:
-
-> ... counting how many of the nucleotides are different from their equivalent in the other string.
+Strategy 2: String Power
+ 
+```ruby 
+    (0..strand1.length).count { |i| strand1[i] != strand2[i] }
+```
 
-translate almost directly into the Ruby code:
+Strategy 3: Each with index
 
 ```ruby
-.count { |char1, char2| char1 != char2 }
+    some_strand.chars.each_with_index.count {|letter, index| letter != other_strand[index] }
```

Comment:
Can we use the variable names `strand1` and `strand2` so the examples remain consistent?

Reply:
I liked that is was more evidently different solutions. But apparently it's not more clear ;-) 
I'll change it. Thanks!